web/templates/support.html

```diff
@@ -17,192 +17,190 @@
   />
   <div class="ui container">
     {{ $ief := ieForm .Gin }}
-    {{ with .Get "users/self/donor_info" }}
-      {{ if .has_donor }}
-        <div class="ui segment">
-          <h1
-            class="ui center aligned header no bottom margin"
-            style="margin: 0 auto; display: table;"
-          >
-            <i class="red heart icon" style="font-size: 90%;"></i>
-            <div class="content">{{ $.T "You are a supporter!" }}</div>
-          </h1>
-          <div class="ui centered subtitle" style="font-size:1.2rem;">
-            {{ $.T "Your Supporter expires <b>%s</b>. No need to feel guilty for not having donated!" (time .expiration) | html }}
-            &lt;3
+    {{ if .Context.User.ID }}
+      {{ with .Get "users/self/donor_info" }}
+        {{ if .has_donor }}
+          <div class="ui segment">
+            <h1
+              class="ui center aligned header no bottom margin"
+              style="margin: 0 auto; display: table;"
+            >
+              <i class="red heart icon" style="font-size: 90%;"></i>
+              <div class="content">{{ $.T "You are a supporter!" }}</div>
+            </h1>
+            <div class="ui centered subtitle" style="font-size:1.2rem;">
+              {{ $.T "Your Supporter expires <b>%s</b>. No need to feel guilty for not having donated!" (time .expiration) | html }}
+              &lt;3
+            </div>
           </div>
-        </div>
+        {{ end }}
```

Comment:
moved the `{{ end }}` from near EOF to here and dedented the whole block below. Also had to change ui a bit with the removal of the `has_donor` check below on previously-L40